data/governments.txt

```diff
@@ -772,19 +502,19 @@ government "Ultaka (Omnis)"
 	fine 0
 
 government "Ultaka Alrul (Omnis)"
-	"display name" "Ultaka (Alrul)"
+	"display name" "Ultaka(Alrul)"
```

Comment:
```suggestion
	"display name" "Ultaka (Alrul)"
```
Wrong, see vanilla.

Reply:
> Everywhere else for the Ultaka though, there is no space. It wouldn't make sense for this to be the only one that has it. I didn't change the others because I thought there might be a reason.

Then those are wrong. This plugin was made back in like 2017~18, it was the first major plugin I've ever released.